### PR TITLE
Update library to use new interface

### LINKED ISSUE #21: Update to use new interface
After the updates to the [protobuf contracts](https://github.com/parallaxsecond/parsec-operations/pull/6) and the subsequent changes to the [interface](https://github.com/parallaxsecond/parsec-interface-rs/pull/28), the test client should be updated to use the new available API.

### PR DESCRIPTION
**Pull request by ionut-arm**
This commit updates the test client library to use the new, updated
interface using the PSA Crypto 1.0 contracts.

Signed-off-by: Ionut Mihalcea <ionut.mihalcea@arm.com>

closes #21

### CODE REVIEW

File: src/stress_test_client.rs [hunk 1]
```diff
@@ -61,12 +62,12 @@ fn generate_string(size: usize) -> String {
...
 enum Operation {
-    CreateDestroyKey,
+    CreatePsaDestroyKey,
     Sign,
     Verify,
-    DestroyKey,
-    ImportDestroyKey,
-    ExportPublicKey,
+    PsaDestroyKey,
+    ImportPsaDestroyKey,
```
Comment (hug-dev) [inline src/stress_test_client.rs]: `PsaImportAndDestroyKey`?

File: src/stress_test_client.rs [hunk 2]
```diff
@@ -61,12 +62,12 @@ fn generate_string(size: usize) -> String {
 
 #[derive(Copy, Clone, Debug)]
 enum Operation {
-    CreateDestroyKey,
+    CreatePsaDestroyKey,
```
Comment (hug-dev) [inline src/stress_test_client.rs]: `PsaCreateAndDestroyKey`?

File: src/stress_test_client.rs (hunk 1 above)
Comment (ionut-arm) [reply src/stress_test_client.rs]: Indeed, shall fix, sorry about that :D

File: src/stress_test_client.rs (hunk 2 above)
Comment (ionut-arm) [reply src/stress_test_client.rs]: Shall fix here as well

**Pull request merged**